### DISCUSSION TITLE
How to get coding habits to work?

### DISCUSSION THREAD

**Discussion by MItCHeLPL** [lowlighter/metrics#478] [Q&A] [answered]
Hi, I can't get coding habits facts to work using both main and latest lowlighter/metrics branch. Access token has full repo access.

yml code:
https://github.com/MItCHeLPL/MItCHeLPL/blob/main/.github/workflows/stats_transparent.yml

output svg:
https://github.com/MItCHeLPL/MItCHeLPL/blob/main/stats_transparent.svg

I tried different setting combinations, but none seem to work

**Comment by lowlighter** [comment, marked as answer]
Hello!

I can't see any errors in your workflow (both files and run) but it has been setup right.

I'll try to debug it in a few days 👍

**Reply by lowlighter** [reply]
This should be fixed by c70b6493c0526f13689b557aa96aa468d1a5c400 👍

**Comment by MItCHeLPL** [comment]
It works now. Thanks!
